core/templates/dev/head/pages/community-dashboard-page/services/translate-text-backend-api.service.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview A service to make http calls present inside servicees related
+ * to community-dashboard-page services
+ */
+
+angular.module('oppia').factory('TranslateTextBackendService', [
```

Comment:
Please add unit tests for this service

Reply:
Don't you think unit tests should be added under a new issue? This issue doesn't address a lack of test coverage, just a code refactoring?

Reply:
@nishantwrp I added the tests.